releasenotes/notes/deprecate-MSGate-class-40495351761c202f.yaml

```diff
@@ -0,0 +1,7 @@
+---
+deprecations:
+  - |
+    The :class:`qiskit.circuit.library.standard_gates.MSGate` has been
+    deprecated and may be removed in a future release. It has been
```

Comment:
```suggestion
    deprecated and moved to `qiskit.circuit.library.generalized_gates`. It has been
```

Reply:
Done in https://github.com/Qiskit/qiskit-terra/pull/4979/commits/29bbc333d01f453f93a4970271eec7091e2fe404